src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2673,14 +2673,43 @@ public ConstructorEdit(INamedTypeSymbol oldType)
-                                                oldSymbol.GetSymbolModifiers() is { IsVirtual: false, IsAbstract: false, IsOverride: false })
+                                            if (oldSymbol.GetSymbolModifiers() is { IsVirtual: false, IsAbstract: false, IsOverride: false } &&
+                                                oldSymbol.ContainingType is { TypeKind: TypeKind.Class or TypeKind.Struct } &&
+                                                !oldSymbol.IsExtern)
                                             {
-                                                semanticEdits.Add(new SemanticEditInfo(editKind, symbolKey, syntaxMap, syntaxMapTree: null, partialType: null, deletedSymbolContainer: containingSymbolKey));
-                                                continue;
+                                                // Deleting an ordinary method is allowed, and we store the newContainingSymbol in NewSymbol for later use
+                                                // We don't currently allow deleting virtual or abstract methods, because if those are in the middle of
+                                                // an inheritance chain then throwing a missing method exception is not expected
+                                                if (oldSymbol is IMethodSymbol
+                                                    {
+                                                        MethodKind:
+                                                            MethodKind.Ordinary or
+                                                            MethodKind.Constructor or
+                                                            MethodKind.EventAdd or
+                                                            MethodKind.EventRemove or
+                                                            MethodKind.EventRaise or
+                                                            MethodKind.Conversion or
+                                                            MethodKind.UserDefinedOperator or
+                                                            MethodKind.PropertyGet or
+                                                            MethodKind.PropertySet
+                                                    })
+                                                {
+                                                    AddDeleteSemanticEdit(semanticEdits, oldSymbol, containingSymbolKey, syntaxMap, cancellationToken);
+                                                    continue;
+                                                }
+                                                else if (oldSymbol is IPropertySymbol propertySymbol)
+                                                {
+                                                    AddDeleteSemanticEdit(semanticEdits, propertySymbol.GetMethod, containingSymbolKey, syntaxMap, cancellationToken);
```

Comment:
>AddDeleteSemanticEdit [](http://example.com/codeflow?start=52&length=21)

Shouldn't the change to TryGetAssociatedMemberDeclaration result in 3 edits being produced for a property symbol without manually adding the accessor deletes?